lib/elixir/lib/module.ex

```diff
@@ -197,7 +197,8 @@ defmodule Module do
 
   As can be seen in the example above, `@doc` and `@typedoc` also accept
   a keyword list that serves as a way to provide arbitrary metadata
-  about the entity. Tools like ExDoc and IEx may use this information to
+  about the entity. Tools like [`ExDoc`](https://hexdocs.pm/ex_doc/)
+  and `IEx` may use this information to
```

Comment:
Minor nitpick: the indentation of this paragraph could be improved :)

Reply:
I will add this to the next commit that takes care of typos and small fixes.
Thank you